packages/cli/src/commands/open.ts

```diff
@@ -24,21 +34,48 @@ export class OpenCommand extends Command {
     description: "The port to serve the reports on. If not set, the server starts on a random port",
   });
 
-  live = Option.Boolean("--live", {
-    description: "Reload pages on any file change in the served directory",
-  });
-
   cwd = Option.String("--cwd", {
     description: "The working directory for the command to run (default: current working directory)",
   });
 
   async execute() {
-    const config = await readConfig(this.cwd, this.config, { output: this.reportDir });
+    const cwd = this.cwd ?? processCwd();
+
+    const isGlobPatternGiven = this.resultsDir.includes("*") || this.resultsDir.includes("?");
+    const summaryFilesGlob = isGlobPatternGiven ? this.resultsDir : join(this.resultsDir, "**", "summary.json");
+    const summaryFiles = await glob(summaryFilesGlob, {
+      mark: true,
+      nodir: false,
+      absolute: true,
+      dot: true,
+      windowsPathsNoEscape: true,
+      cwd,
+    });
+    let config: FullConfig;
+
+    // there's no generated report to serve, so we generate it first in a temp directory
+    if (!summaryFiles.length) {
+      config = await readConfig(cwd, this.config, {
+        port: this.port,
+        output: join(cwd, ".allure", randomUUID()),
```

Comment:
Let's use mkdtemp + temp and remove the directory after the serving is done by ctrl+c (reports may easily weigh a hundred MB; 10 opens and we've eaten a gig...)